src/stories/Table.stories.tsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Table } from "@components/Table";
+import { TableHeaderCell } from "@components/TableHeaderCell";
+import { Tabs } from "@components/Tabs";
+import { TabPane } from "@components/TabPane";
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+	title: "Table",
+	component: Table,
+} as ComponentMeta<typeof Table>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof Table> = (args) => <Table {...args} />;
+
+const defaultArgs = {
+	dataSource: [
+		{
+			key: 0,
+			name: "a55 Debenture Series-1",
+			amount: "0.22 USDC",
+			date: "02.11.21",
+		},
+		{
+			key: 1,
+			name: "a55 Debenture Series-1",
+			amount: "0.22 USDC",
+			date: "02.11.21",
+		},
+		{
+			key: 2,
+			name: "a55 Debenture Series-1",
+			amount: "0.22 USDC",
+			date: "02.11.21",
+		},
+		{
+			key: 3,
+			name: "a55 Debenture Series-1",
+			amount: "0.22 USDC",
+			date: "02.11.21",
+		},
+		{
+			key: 4,
+			name: "a55 Debenture Series-1",
+			amount: "0.22 USDC",
+			date: "02.11.21",
+		},
+	],
+	columns: [
+		{
+			title: () => <TableHeaderCell label="Name" icon="stacked-column-down" />,
```

Comment:
what's your opinion on hiding having to do this?
we can have our own column object definition that has title and optional icon and then perform a map in the table component itself replacing those with what you are doing here?

Reply:
I want a clean simple api and I feel like this is just something you have to know/think but it's just default behaviour. we can always allow a function to be passed to let the user render their own header like ant design allows

Reply:
Yes, that's definitely something we can do. I agree that it'd make things easier, as a tableheader without icon would probably be the exception rather than the rule.